test/perf/merge-nested.js

```diff
@@ -35,6 +36,14 @@ function merge(n, create) {
   return s;
 }
 
+function mergeRxjs6(n, create) {
+  var s = [];
+  for(var i=0; i<n; ++i) {
+    s.push(create(a));
+  }
+  return create(a).pipe(rxjs6Operators.merge(...s));
```

Comment:
I don't think this is testing the same intent, and I apologize if that intent wasn't clear!  The intent is specifically to merge pairs of streams, rather than to rely on a library's bulk merge operator.

The reason is that across a large application, you may not tend to have all the streams in your hand at once.  For example, you might call a function which, unbeknownst to you, internally merges 2 streams, and returns the result to you, and you then merge that with another stream, and so on.

MostJS focuses not only on performance, but performance scalability, ie how does it perform as usage scales _without_ the dev having to think about it.  For example, MostJS can (and does) optimize the situation described above.

So, I think rx 6 can reuse the `merge` helper function above, and provide a small wrapper lambda around `rxjs6.from` as `create`, similar to what the rx 5 tests do.

Does that make sense?